rules/windows/process_creation/process_creation_win_manipulation_of_tokens_via_impersonate_binary.yml

```
@@ -0,0 +1,29 @@
+title: Possible Manipulation Of Tokens on a Windows computers remotely Detected via Impersonate
```

Since the logic of the rule is focused on the execution and not the impersonation. I think we should call it
```suggestion
title: Impersonate Execution
```